components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/nashorn/JsNashornAuthenticationContext.java

```diff
@@ -174,6 +173,22 @@ protected String getAuthenticatedIdPOfCurrentStep() {
 
     }
 
+    protected String getAuthenticatedAuthenticatorOfCurrentStep() {
+
+        if (getContext().getSequenceConfig() == null) {
+            //Sequence config is not yet initialized
+            return null;
+        }
+
+        StepConfig stepConfig = getContext().getSequenceConfig().getStepMap()
+                .get(getContext().getCurrentStep());
+        if (stepConfig != null) {
+            return stepConfig.getAuthenticatedAutenticator().getName();
+        }
+        return null;
+
```

Comment:
remove empty lines

Reply:
check all the places

Reply:
Resolved by https://github.com/wso2/carbon-identity-framework/pull/4662/commits/c0d6dbd72bfc8d0e4ec327f365e304b2cb9051de